skins/laika/templates/Base_Layout.html.twig

```diff
@@ -39,7 +39,10 @@
 {% endblock %}
 
 {% block scripts %}
-    {% if not assets_path %}
+    {% if assets_path %}
+        <script src="{% if assets_path %}{$ assets_path $}{% else %}{$ basepath $}/skins/laika{% endif %}/js/chunk-vendors.js"></script>
```

Comment:
why check again for assets_path?

Also, in the else branch, we need to add chunk-common